mlir/include/mlir/Analysis/Presburger/PresburgerRelation.h

```diff
@@ -213,6 +213,10 @@ class PresburgerRelation {
   /// also be a union of convex disjuncts.
   PresburgerRelation computeReprWithOnlyDivLocals() const;
 
+  /// Simplify each disjunct, see IntegerRelation::simplify for details. If
+  /// disjunct is empty it will not be merged into the new set.
```

Comment:
You don't need to mention the implementation details. Only the first sentence is enough:

"Simplify each disjunct, canonicalizing each disjunct and removing redundencies"